src/mock/waypoint.js

```diff
@@ -8,6 +8,29 @@ const MAX_PIC = 10;
 const MIN_PRICE = 10;
 const MAX_PRICE = 1000;
 
+const dateMocksArray = [
```

Comment:
```suggestion
const dateMocks = [
```
имя переменной не должно содержать тип